content/agent/faq/downgrade-datadog-agent.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Downgrade the Agent to a prior minor version
+kind: faq
+further_reading:
+---
+
+For DEB or RPM package of the Datadog Agent, find below instructions to downgrade the Datadog Agent to a version `6.<VERSION>`.
+
+**Note**: Those instructions works only for Datadog Agent version 6.x and above.
+
+## Debian/Ubuntu
+
+### CLI
+
+```
+sudo apt-get update && sudo apt-get install --force-yes datadog-agent=1:6.<VERSION>
```

Comment:
The correct version specifier is `datadog-agent=1:6.Y.Z-1` (note the `-1` suffix)

where `6.Y.Z` is an Agent version, for example `6.5.1`